### PR TITLE
Remove prettier config from repo

### PR DESCRIPTION
**Pull request by vgoh1**
# Description 
Contrary to branch name, I decided to get rid of the prettier config entirely since I couldn't get it to conform to airbnb's style guide, plus this way instead of following two different styles we'll follow one.

# Type of change 
- [ ] Bug fix
- [ ] New feature
- [x] Refactoring
- [ ] Documentation 

# Problem
Prettier was conflicting with airbnb style guide so after you run prettier, npm run lint will complain. Also needed to double check whether prettier was causing some formatting issues to be ignored such as missing spaces. (Problem was raised in PR https://github.com/Researchify/Researchify/pull/198. [See run to see it throwing errors for formatting](https://github.com/Researchify/Researchify/runs/3479869805?check_suite_focus=true))

# Solution description
See title. Also had to reformat a lot of files as a result.

# Tests 
Backend, frontend, scholly base should run and work without issues.

# Relevant document/ link (if any) 
Any related documents could help reviewers to review this PR 
You'll want to configure your VSCode to use eslint rather than prettier, instructions: https://daveceddia.com/vscode-use-eslintrc/

# Risk
Identity the risk level (low, medium, high) and indicate which component(s) will be affected if bugs exist.
High

### CODE REVIEW

File: client/src/actions/publications.js [hunk 1]
```diff
@@ -73,75 +73,73 @@ export const updatePublication = (id, publication) => async (dispatch) => {
...
-      ) {
-        // no new publications retrieved but not end of profile
+export const importPublications = (author_id, startFrom, teamId) => async (dispatch) => {
+  try {
+    dispatch({
+      type: IMPORT_REQUEST,
+    });
+    const result = await api.importPublications(author_id, startFrom, teamId);
+    const pageNo = startFrom / pageSize + 1;
+    if (result.data.reachedEnd === true) {
```
Comment (aksh0001) [inline client/src/actions/publications.js:83]: remove `=== true`

**Review by aksh0001 — APPROVED** [review]
LGTM, these changes seem to comply with the style guide better!

File: client/src/actions/publications.js (hunk 1 above)
Comment (vgoh1) [reply client/src/actions/publications.js]: will do

**Review by jfan0010 — APPROVED** [review]
LGTM

**Review by pmyip3 — APPROVED** [review]

**Pull request merged**